app/models/broadcast_announcement.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: broadcast_announcements
+#
+#  id              :bigint           not null, primary key
+#  expiry          :date
+#  link            :text
+#  message         :text
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint           not null
```

Comment:
We should allow nulls here so that in the future we can have admin announcements.